nucypher/cli/actions.py

```diff
@@ -278,3 +277,25 @@ def make_cli_character(character_config,
         console_emitter(message="WARNING: Running in Federated mode", color='yellow')
 
     return CHARACTER
+
+
+def select_stake(stakeholder) -> Stake:
+    enumerated_stakes = dict(enumerate(stakeholder.stakes))
+    painting.paint_stakes(stakes=stakeholder.stakes)
+    choice = click.prompt("Select Stake",
+                          type=click.IntRange(min=0, max=len(enumerated_stakes)))
```

Comment:
Is the `max` parameter for `IntRange` inclusive? - in which case we should use `len(enumerated_stakes)-1` as the max.

Alternatively, how are the choices numbered in the prompt? Do they start at [0] or [1]? Depending, we could also do
```python
choice = click.prompt("Select Stake",
                          type=click.IntRange(min=1, max=len(enumerated_stakes)))
chosen_stake  = enumerated_stakes[choice-1]
```

Reply:
Choices are 0-indexed.  Yeah It needs the `-1` Good eye.

Reply:
:heavy_check_mark: